PWGCF/GenericFramework/Tasks/flowGenericFramework.cxx

```diff
@@ -718,127 +722,127 @@ struct GenericFramework {
       if (!mcParticle.isPhysicalPrimary())
         return;
       if (cfgFillQA)
-        FillTrackQA<kReco, kBefore>(track, vtxz);
+        fillTrackQA<kReco, kBefore>(track, vtxz);
 
       if (mcParticle.eta() < etalow || mcParticle.eta() > etaup || mcParticle.pt() < ptlow || mcParticle.pt() > ptup || track.tpcNClsFound() < cfgNcls)
         return;
 
       if (cfgUseAdditionalTrackCut && !trackSelected(track, field))
         return;
 
-      int pid_index = 0;
+      int pidIndex = 0;
       if (cfgUsePID) {
         if (mcParticle.pdgCode() == 211)
-          pid_index = 1;
+          pidIndex = 1;
         if (mcParticle.pdgCode() == 321)
-          pid_index = 2;
+          pidIndex = 2;
         if (mcParticle.pdgCode() == 2212)
-          pid_index = 3;
```

Comment:
Please, for the next iteration, use named constants from `TPDGCode.h` instead of numbers